tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -12,15 +14,69 @@
 from sentry.testutils.helpers.datetime import before_now, iso_format
 
 
-class TestIdentfiyStacktracePaths(TestCase):
+class BaseDeriveCodeMappings(TestCase):
```

Comment:
On A future PR, I will make the Python group of tests also inherit from this class.